f8a_worker/dispatcher/nodes.yml

```diff
@@ -284,6 +284,12 @@
       max_retry: 0
       storage: 'BayesianPostgres'
       queue: '{DEPLOYMENT_PREFIX}_{WORKER_ADMINISTRATION_REGION}_GitHubManifestMetadataGithubTask_v0'
+    - name: 'github_manifest_details'
+      classname: 'GithubStats'
+      import: 'f8a_worker.workers'
```

Comment:
You will need to place the following line to `f8a_worker/workers/__init__.py` to make this import work:
```python
from f8a_worker.workers.githubstats import GithubStats
```

But I would suggest to place this logic to `github_details` task that we already have (see bellow).

Reply:
removed the new task 'github_manifest_details' from node.yml and appended the logic with  'github_details' task.